### PR TITLE
Allow insecure http for SDK create_drone_client [FLYW-1259]

### PR DESCRIPTION
**Pull request by ehlertjd**
Updated usage:
```
fw = flywheel.create_drone_client(..., port=8080, _force_insecure=True)
```

### Review Checklist

- Tests were added to cover all code changes
- Is there a DB upgrade or fix?
  - If so is it covered by integeration tests and run against dev?
- Would this PR benefit from a test plan? If so, is one provided?
- Documentation was added / updated
- Code and tests follow standards in CONTRIBUTING.md
- Extra care is taken and log messages are created for [Critical Data](https://github.com/flywheel-io/core/wiki/Critical-Data-Flows)

### CODE REVIEW

File: sdk/src/python/tests/integration_tests/sdk_test_case.py [hunk 1]
```diff
@@ -32,12 +32,42 @@ def get_api_key():
 
     return api_key, device_key
 
+def make_drone_client():
+    if init_db.SCITRAN_PERSISTENT_DB_URI:
```
Comment (roycollins) [inline sdk/src/python/tests/integration_tests/sdk_test_case.py]: I assumed this has to be set for the tests to run. Im just curious when this is not true.

Comment (ehlertjd) [reply sdk/src/python/tests/integration_tests/sdk_test_case.py]: I will occasionally run the SDK tests directly against my running local dev environment by setting the `SdkTestKey` to an api-key, which is when this clause is inverted. I'm sure that we could devise a better way to do low-latency iterations given sufficient time and motivation.

File: sdk/src/python/tests/integration_tests/sdk_test_case.py [hunk 2]
```diff
@@ -32,12 +32,42 @@ def get_api_key():
...
+
+
 def make_clients():
     api_key, device_key = get_api_key()
 
     fw = flywheel.Flywheel(api_key)
     if device_key:
         fw_device = flywheel.Flywheel(device_key)
+    else:
+        fw_device = None
```
Comment (hkethi002) [inline sdk/src/python/tests/integration_tests/sdk_test_case.py]: Is there a reason we're using device and drone, I know that we use them interchangeably in core, but this maybe a larger topic of keeping names and variables consistent.

File: sdk/codegen/src/main/resources/fw-python/drone_login.py [hunk 3]
```diff
@@ -20,7 +20,11 @@ def create_drone_client(host, secret, method, name, port=443, **kwargs):
         util.set_verify_ssl(session)
 
         # Get auth status to determine device id
-        base_uri = 'https://{}:{}/api'.format(host, port)
+        force_insecure = kwargs.pop('_force_insecure', False)
+        if force_insecure:
```
Comment (ryandavisflywheel) [inline sdk/codegen/src/main/resources/fw-python/drone_login.py]: `assert(type(force_insecure) == bool)` 
here so it blows up for anything else? ... maybe I'm being paranoid, but a
`force_insecure = "False"` 
among others vals/types for force_insecure are truthy

File: sdk/src/python/tests/integration_tests/sdk_test_case.py (hunk 2 above)
Comment (ehlertjd) [reply sdk/src/python/tests/integration_tests/sdk_test_case.py]: Good point - I'm going to leave it for now, but maybe a refactor would be in order if we ever start making the distinction clearer.

**Pull request merged**